### PR TITLE
Add brief summary of possible ways to use ESMValTool

### LINKED ISSUE #157: Add a note to installation episode about central installation of ESMValTool on CEDA-JASMIN
Thanks to @valeriolucarini and @agstephens.  esmvaltool is installed centrally on CEDA/JASMIN, so you can use it now without having to install it yourselves. 
In the tutorial, a note should be added to episode installation.

### PR DESCRIPTION
**Pull request by rswamina**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [x] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [ ] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #157

### CODE REVIEW

**Comment by rswamina** [comment]
@SarahAlidoost - Were you going to add a few lines on downloading the source files and compiling them (not with Github) in the Development mode lesson? Could you do that so I can link to it here and request a review?

File: _episodes/02-installation.md [hunk 1]
```diff
@@ -25,6 +25,31 @@ be found in the
...
 
+
+> ## ESMValTool Installation/Usage  Options - A quick summary
+> Before we begin, here are all the possible ways in which you can use ESMValTool
+> depending on your level of expertise or involvement with ESMvalTool and 
+> associated software such as GitHub and Conda.
+>
+> 1) If you have access to a server where ESMValTool is already installed 
+> as a module, for e.g., the ``CEDA JASMIN`` server, you can simply
+> load the module and start using ESMValTool with the following :
```
Comment (SarahAlidoost) [inline _episodes/02-installation.md:36]:
```suggestion
> load the module with the following:
```

File: _episodes/02-installation.md [hunk 2]
```diff
@@ -25,6 +25,31 @@ be found in the
...
+> ```bash
+> module load esmvaltool
+> ```
```
Comment (SarahAlidoost) [inline _episodes/02-installation.md:39]:
```suggestion
> ```

After loading ``esmvaltool``, we can start using ESMValTool. Please see the [next lesson]({{ page.root }}{% link _episodes/03-configuration.md %}). 
```

File: _episodes/02-installation.md [hunk 3]
```diff
@@ -25,6 +25,31 @@ be found in the
...
+> 2) If you would like to install ESMValTool as a conda package, then this lesson 
+> will tell you how!
+> 
+> 3) If you are already familiar with ESMValTool and would like to start experimenting
+> with existing diagnostics, you can dowload the source code from the ESMvalTool Github
+> page, compile and run as given here (link missing).
+>
```
Comment (SarahAlidoost) [inline _episodes/02-installation.md:46]:
```suggestion
```

File: _episodes/02-installation.md [hunk 4]
```diff
@@ -25,6 +25,31 @@ be found in the
...
+> 4) If you would like to start contributing to ESMvalTool, please see the instructions for 
+> source installation in the lesson [Development and
```
Comment (SarahAlidoost) [inline _episodes/02-installation.md:48]:
```suggestion
> 3) If you would like to start experimenting with existing diagnostics or contributing to ESMvalTool, please see the instructions for 
> source installation in the lesson [Development and
```
```suggestion
> 4) If you would like to start contributing to ESMvalTool, please see the instructions for 
> source installation in the lesson [Development and
```

File: _episodes/02-installation.md [hunk 5]
```diff
@@ -25,6 +25,31 @@ be found in the
...
+> module load esmvaltool
+> ```
+> 2) If you would like to install ESMValTool as a conda package, then this lesson 
+> will tell you how!
+> 
+> 3) If you are already familiar with ESMValTool and would like to start experimenting
+> with existing diagnostics, you can dowload the source code from the ESMvalTool Github
+> page, compile and run as given here (link missing).
+>
+> 4) If you would like to start contributing to ESMvalTool, please see the instructions for
```
Comment (SarahAlidoost) [inline _episodes/02-installation.md:47]:
```suggestion
> 3) If you would like to start experimenting with existing diagnostics or contributing to ESMvalTool, please see the instructions for 
```

**Review by SarahAlidoost — CHANGES_REQUESTED** [review]
@rswamina thank you for improving the lesson. I added a few comments. I merged numbers 3 and 4 into one item. Because they only differ on how to get the source code i.e downloading zip file or using git clone. The installation procedure is the same. I will explain this in the episode "development and contribution". 
In addition, I think it is not necessary to have these items in a box. Could you please remove the {: .solution} format and convert these items to normal texts?

**Comment by rswamina** [comment]
@SarahAlidoost - I can do that and also I want to add a link to the JASMIN module load option. Will notify when I finish and then it should be ready to merge?

**Comment by rswamina** [comment]
@SarahAlidoost  - Have made the suggested changes and added link to the JASMIN page. This hsould be ok to merge now unless you see something.

File: _episodes/02-installation.md [hunk 6]
```diff
@@ -25,6 +25,30 @@ be found in the
 We will first install Conda, and then ESMValTool. We end this chapter by testing
 that the installation was successful.
 
+
+## ESMValTool Installation/Usage  Options - A quick summary
```
Comment (SarahAlidoost) [inline _episodes/02-installation.md:29]:
```suggestion
```

File: _episodes/02-installation.md [hunk 7]
```diff
@@ -25,6 +25,30 @@ be found in the
...
+ Before we begin, here are all the possible ways in which you can use ESMValTool
+ depending on your level of expertise or involvement with ESMvalTool and 
+ associated software such as GitHub and Conda.
```
Comment (SarahAlidoost) [inline _episodes/02-installation.md:32]:
```suggestion
Before we begin, here are all the possible ways in which you can use ESMValTool
depending on your level of expertise or involvement with ESMvalTool and 
associated software such as GitHub and Conda.
```

File: _episodes/02-installation.md [hunk 8]
```diff
@@ -25,6 +25,30 @@ be found in the
...
+
+ 1) If you have access to a server where ESMValTool is already installed 
+ as a module, for e.g., the [CEDA JASMIN](https://help.jasmin.ac.uk/article/4955-community-software-esmvaltool)
```
Comment (SarahAlidoost) [inline _episodes/02-installation.md:35]:
```suggestion
1. If you have access to a server where ESMValTool is already installed
as a module, for e.g., the [CEDA JASMIN](https://help.jasmin.ac.uk/article/4955-community-software-esmvaltool)
```

File: _episodes/02-installation.md [hunk 9]
```diff
@@ -25,6 +25,30 @@ be found in the
...
+
+ 1) If you have access to a server where ESMValTool is already installed 
+ as a module, for e.g., the [CEDA JASMIN](https://help.jasmin.ac.uk/article/4955-community-software-esmvaltool) 
+server, you can simply load the module with the following:
+ ```bash
+ module load esmvaltool
+ ```
+
+After loading ``esmvaltool``, we can start using ESMValTool. Please see the [next lesson]({{ page.root }}{% link _episodes/03-configuration.md %}). 
+
```
Comment (SarahAlidoost) [inline _episodes/02-installation.md:42]:
```suggestion
~~~bash
module load esmvaltool
~~~
After loading ``esmvaltool``, we can start using ESMValTool. Please see the [next lesson]({{ page.root }}{% link _episodes/03-configuration.md %}). 
```

File: _episodes/02-installation.md [hunk 10]
```diff
@@ -25,6 +25,30 @@ be found in the
...
+After loading ``esmvaltool``, we can start using ESMValTool. Please see the [next lesson]({{ page.root }}{% link _episodes/03-configuration.md %}). 
+ 
+2) If you would like to install ESMValTool as a conda package, then this lesson 
+ will tell you how!
+ 
+ 3) If you would like to start experimenting with existing diagnostics or contributing to ESMvalTool, please see the instructions for 
+ source installation in the lesson [Development and
+ contribution]({{ page.root }}{% link _episodes/08-development-setup.md %}) and 
+ in the [documentation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html).
+
```
Comment (SarahAlidoost) [inline _episodes/02-installation.md:50]:
```suggestion
2. If you would like to install ESMValTool as a conda package, then this lesson
will tell you how!
3. If you would like to start experimenting with existing diagnostics or contributing to ESMvalTool, please see the instructions for 
source installation in the lesson [Development and
contribution]({{ page.root }}{% link _episodes/08-development-setup.md %}) and
in the [documentation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html).
```

**Comment by SarahAlidoost** [comment]
> @SarahAlidoost - Have made the suggested changes and added link to the JASMIN page. This hsould be ok to merge now unless you see something.

@rswamina thank you for addressing my comments. I suggested some changes to fix the style. Could you please have a look?

**Comment by rswamina** [comment]
@SarahAlidoost - I am happy with the changes in formatting. Thanks!

**Comment by SarahAlidoost** [comment]
> @SarahAlidoost - I am happy with the changes in formatting. Thanks!

Could you please commit the suggestions?

**Comment by rswamina** [comment]
> > @SarahAlidoost - I am happy with the changes in formatting. Thanks!
> 
> Could you please commit the suggestions?

Done! Anything else?

**Comment by SarahAlidoost** [comment]
> > > @SarahAlidoost - I am happy with the changes in formatting. Thanks!
> > 
> > 
> > Could you please commit the suggestions?
> 
> Done! Anything else?

Awesome, no.

**Review by SarahAlidoost — APPROVED** [review]

**Pull request merged**